Sources/Markdown/Base/RawMarkup.swift

```diff
@@ -24,7 +24,7 @@ enum RawMarkupData: Equatable {
     case thematicBreak
     case htmlBlock(String)
     case listItem(checkbox: Checkbox?)
-    case orderedList
+    case orderedList(start: Int?)
```

Comment:
Since this is an API breaking change, this will require a coordinated change with Swift-DocC. Is there a way we could introduce the change without requiring the coordination?

Reply:
Also, I think this could benefit from a default value (`orderedList(start: Int? = nil)`)

Reply:
The `RawMarkup` type isn't used in Swift-DocC, so this won't cause source breakage there. The public accessor to the start index is the new `start` property on the `OrderedList` type.

Reply:
Looking at this again, this type (`RawMarkupData`) isn't public API. Other enum cases don't use a default value; that's enforced via the public `Markup` types which wrap `RawMarkup`. I don't think it would be useful to include a default value for the list start value here; that comes from cmark.